profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -83,8 +83,9 @@ function nexteuropa_varnish_cache_purge_rule_label($purge_rule) {
  */
 function nexteuropa_varnish_workbench_moderation_transition($node, $previous_state, $new_state) {
   $purge
-    = ($new_state == workbench_moderation_state_published()) ||
-      ($previous_state == workbench_moderation_state_published() && $new_state !== $previous_state);
+    = ($node->workbench_moderation['my_revision']->published) ||
```

Comment:
I don't understand why we need to check the workbench_moderation variable inside the node, are not the variables provided in the hook enough?
If I'm not wrong, the only moment where a PURGE request needs to be sent is when the $new_state is published (this is the moment when the publicly available content is changed)